queue/src/main/java/com/google/common/eventbus/AllowConcurrentEvents.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2007 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.common.eventbus;
```

Comment:
I tend to this, and unfortunately, I'm afraid this is the only "easy" approach.

Yesterday I'm work with removing all guava-related classes in `killbill-queue`. But even after remove all guava classes, I'm still unable to pass the build, and maven-dependency-plugin keep complaining about:

> Used undeclared dependencies found:
>   [WARNING]    com.google.guava:guava:jar:31.1-jre:compile

I guess this is because we keep the package name same. I'm planning to test this by:

1. temporarily move all eventbus class to different package and see the result,
2. take a look at how `killbill-queue` is different with `killbill-jdbi` (as both projects forked another project in the same package name)

But if we can just change the package name to something other than `com.google.common.eventbus` (I'm thinking `org.killbill.bus`, same package as `DefaultPersistentBus`), maybe it would be easier? I know that there's chances that I'm miss something, but again I can't see any disadvantage by moving package name.

What do you think @sbrossie ?

Reply:
Yeah I confirm that simply move to another package fix the maven-dependency-plugin problem I explained above.

Reply:
Yes I think this make sense: Rename the package completely, perhaps `org.killbill.commons.queue` ?

Reply:
Hm.. now I prefer `org.killbill.eventbus`, since this is "our" event bus. Or if "commons" should exist , then `org.killbill.commons.eventbus`. IMO ended it with `queue` will be weird since none of classes talking about "queue", and we have another package named "queue" that host `DBBackedQueue`, `QueueEvent` (in `queue.api`), etc. Thought?

Reply:
Good suggestion: Let's go with `org.killbill.commons.eventbus` - not that we already have some `org.killbill.commons.xx` packages e.g [here](https://github.com/killbill/killbill-commons/blob/master/skeleton/src/main/java/org/killbill/commons/skeleton/modules/BaseServerModule.java#L20), but I know we have not been consistent e.g [here](https://github.com/killbill/killbill-commons/blob/master/clock/src/main/java/org/killbill/clock/Clock.java#L20).